src/i18n/i18n.ts

```diff
@@ -608,6 +610,7 @@ const dict = {
         browsernotification_tooltip:
             "Activar o desactivar notificaciones del navegador",
         browsernotification_error: "Permisos de notificación denegados",
+        invalid_pair: "Par no válido",
```

Comment:
@kilrau did you check these?

Reply:
Yes. ES was ok, for CN I did a small fix: https://github.com/BoltzExchange/boltz-web-app/pull/551/commits/dcfce3a8bbc6ddb28e6c20588aa08b7b43338d59